.github/workflows/gcp_dev_deploy.yaml

```diff
@@ -0,0 +1,64 @@
+name: Deploy on Cloud Run for clients
+
+on:
+  push:
+    branches:
+      - release
+
+env:
+  PROJECT_ID: ${{ secrets.GCP_PROJ_ID }}
+  RUN_REGION: europe-west1
+  IMAGE_NAME: release-img
+  GCP_REPO: eu.gcr.io
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      # Setup gcloud CLI
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '286.0.0'
+          service_account_email: ${{ secrets.GCP_EMAIL }}
+          service_account_key: ${{ secrets.GCP_KEY }}
+
+      # Configure gcloud CLI
+      - name: gcloud set up
+        run: |
+          gcloud config set project $PROJECT_ID
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Build with Gradle
+        working-directory: project
+        run: ./gradlew assemble
+
+      - name: Rebuild docker image
+        working-directory: project
+        run: docker build -t $IMAGE_NAME .
+
+      - name: Configure docket authn
+        working-directory: project
+        run: gcloud auth configure-docker
+
+      - name: Tag image
+        working-directory: project
+        run: docker tag $IMAGE_NAME $GCP_REPO/$PROJECT_ID/$IMAGE_NAME:$GITHUB_SHA
+
+      - name: Push image
+        working-directory: project
+        run: docker push $GCP_REPO/$PROJECT_ID/$IMAGE_NAME:$GITHUB_SHA
+
+      - name: Deploy to cloud run
```

Comment:
What is the goal for this PR. I was assuming that we are not going to use cloud run, and use a VM with an LB instead.